public/components/marketing-pricings/2.html

```diff
@@ -0,0 +1,148 @@
+<div class="mx-auto max-w-4xl px-4 py-8 sm:px-6 sm:py-12 lg:px-8">
+    <div class="grid grid-cols-1 gap-4 sm:grid-cols-2 lg:grid-cols-3 sm:items-center md:gap-8">
+        <div class="border-2 rounded bg-white shadow">
```

Comment:
I don't like the `border-2` here and the `shadow`, let's try `border border-gray-100 shadow-sm`.